### ISSUE TITLE
4.11.2 Successfully opens News but not Tedit (Upgrading from 4.2.2)

### ISSUE THREAD

**Issue by SuperCode747** [TEdit/Terraria-Map-Editor#1689]
# READ THIS FIRST
- **Sprite Issues?** See https://github.com/TEdit/Terraria-Map-Editor/wiki/Install-Requirements#for-sprite-viewing
- **Won't Start?** See https://github.com/TEdit/Terraria-Map-Editor/wiki/Troubleshooting#application-launch-or-map-loading-issue
- **Map not loading?** See https://github.com/TEdit/Terraria-Map-Editor/wiki/Troubleshooting#application-launch-or-map-loading-issue

# If the Wiki did not help, fill out this form completely! If you are missing steps to reproduce your issue will be closed. Thank you.

**Before Submitting an Issue**
* If you are having issues loading a world, please attach it to the current version world thread (see pinned issues).
* For a crash, please check the log.

**Describe the bug**
What is the problem.

So i downloaded Tedit 4.2.2 before, and i wanted to update it to the latest version. It prompted me to upgrade. Got the zip file and unzipped it. Clicked on Tedit, and it only opened News. Closed news, then clicked Tedit again. Brief popup that said "Unhandled Exception Occurred" for about a second. ~~Later when i tried this to reproduce it, it does not show the error popup.~~

EDIT: This is reproducible, but you probably won't get the popup, or you will for only about a split second. I cannot open Tedit 4.11.2 because of this error.

This is probably a crash, ~~but there is no crash log to go off of, so i cannot check the log to see what happened.~~
Edit: See one of my comments, im new to this whole troubleshoot thing and didnt know where the logs were. Now i know.

When i got the Zip file, a schematics folder was outside of the Tedit folder. I wonder why that was... Dragged it inside the folder and got the popup for longer than a brief flash.

1. Unzip Tedit
2. Go to Tedit Folder
3. Click Tedit.exe to open it, if you get the "Dangerous App" warning, close it by running the app anyways.
5. Click close on news page
6. Click Tedit (News pops up)
7. Tedit does not open.
8. For best results, try dragging your schematics outside/inside the folder? idk.

**Expected behavior**
It would open news, i would click close, then Tedit window would pop up with the editor, and i would edit my world in 4.11.2. The world was taken off my Android, so my previous install wouldn't work for that reason.

**Screenshots**
~~There are no screenshots of the error because of how fast it happened. i have attached a screenshot of the news because of the file requirement.~~
EDIT: Finally was fast enough for a screenshot!

> ![The error that happened is an Unhandled Exception](https://user-images.githubusercontent.com/62219099/194169416-804c7260-2516-421e-b423-ebfea316875f.jpeg)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version: Tedit 4.11.2, with 4.2.2 installed
 - Terraria Version: Not applicable, Latest Terraria if you need that info. (Mobile 1.4.4.1)

5. Click close on news page
6. Click Tedit (News pops up)
7. Tedit does not open.
8. For best results, try dragging your schematics outside/inside the folder? idk.

**Expected behavior**
It would open news, i would click close, then Tedit window would pop up with the editor, and i would edit my world in 4.11.2. The world was taken off my Android, so my previous install wouldn't work for that reason.

**Screenshots**
~~There are no screenshots of the error because of how fast it happened. i have attached a screenshot of the news because of the file requirement.~~
EDIT: Finally was fast enough for a screenshot!

> ![The error that happened is an Unhandled Exception](https://user-images.githubusercontent.com/62219099/194169416-804c7260-2516-421e-b423-ebfea316875f.jpeg)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version: Tedit 4.11.2, with 4.2.2 installed
 - Terraria Version: Not applicable, Latest Terraria if you need that info. (Mobile 1.4.4.1)

**Comment by RussDev7** [comment]
1. Try reinstalling Terraria.
2. Delete `AppData\Roaming\TEdit`.
3. Install TEdit to a fresh directory.
Any of these work to resolve this? I've had to do these in the following to resolve this.

**Comment by SuperCode747** [comment]
1. My terraria install works fine. I only included that bit of information because i saw in the changelog that Tedit 4.2.2 does not support mobile android .wld files.
2. I deleted that file from my system, Tedit is still not launching other than displaying News, and there is no crash error (yet)
3. I got the zip file, unzipped, and clicked on Tedit, it still does the thing where it just displays News and does not open.

**Comment by BinaryConstruct** [comment]
Can you please check your log file and post it here? https://docs.binaryconstruct.com/advanced/troubleshooting#step-1-check-the-error-log thanks!

**Comment by SuperCode747** [comment]
The bug has been fixed! Before and after photos of my task manager prove this.
Before:
![tskmgr](https://user-images.githubusercontent.com/62219099/194205358-3a25551e-34da-419d-9c56-b26c2a93dfc3.jpg)
After:
![tskmgrFixed](https://user-images.githubusercontent.com/62219099/194205409-0a03e7a2-1a3e-4b82-8fd1-ce8951d73386.jpg)

**Comment by RussDev7** [comment]
> In answer to your comment, it still doesn't work right, and still launches News then quits.
> 
> > 1. Try reinstalling Terraria.
> 
> My terraria install works fine. I only included that bit of information because i saw in the changelog that Tedit 4.2.2 does not support editing mobile android .wld files, and i wished to upgrade to 4.11.2 because i wanted to edit a world from my mobile device
> 
> > 2. Delete `AppData\Roaming\TEdit`.
> 
> I tried that when i saw your comment, the error is gone, but it's still not opening Tedit. Edit: Opened wrong folder by mistake, trying now.
> 
> > 3. Install TEdit to a fresh directory.
> 
> It is a fresh new zip file of Tedit 4.11.2 from the GitHub. I extracted it from the zip, which my OS already creates a fresh directory for. Then i went into the folder, where the Tedit resides and clicked on it. News popped up yet again and still didn't open TEdit.
> 
> > Any of these work to resolve this? I've had to do these in the past to resolve this.
> 
> I tried your solutions, it still does the thing where it just displays News and does not open, but my cursor has that blue spinner for a second and then stops.
> 
> TLDR, thanks for the info, but it still does not work.

If only tried my first suggestion this would have been closed a long time ago haha.
